interfaces/IERC20.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.24;
+
+interface IERC20 {
```

Comment:
See my [latest comment](https://github.com/gnosis/labs-contracts/pull/1#discussion_r1648033593) - please import this from the open-zeppelin dependency package.